edx_exams/apps/core/admin.py

```diff
@@ -19,4 +25,40 @@ class CustomUserAdmin(UserAdmin):
     )
 
 
+class ProctoringProviderAdmin(admin.ModelAdmin):
+    """ Admin configuration for the Proctoring Provider model """
+    list_display = ('name', 'verbose_name', 'lti_configuration_id')
+    search_fields = ('name', 'verbose_name', 'lti_configuration_id')
+    ordering = ('name',)
+
+
+class ExamAdmin(admin.ModelAdmin):
+    """ Admin configuration for the Exam model """
+    list_display = ('resource_id', 'course_id', 'provider', 'content_id', 'exam_name',
+                    'exam_type', 'time_limit_mins', 'due_date', 'hide_after_due', 'is_active')
+    search_fields = ('resource_id', 'course_id', 'provider__name', 'content_id', 'exam_name',
+                     'exam_type', 'time_limit_mins', 'due_date', 'hide_after_due', 'is_active')
+    ordering = ('exam_name',)
+
+
+class ExamAttemptAdmin(admin.ModelAdmin):
+    """ Admin configuration for the Exam Attempt model """
+    list_display = ('user', 'exam', 'attempt_number', 'status', 'start_time',
+                    'allowed_time_limit_mins')
+    search_fields = ('user__username', 'exam__name', 'attempt_number', 'status', 'start_time',
+                     'allowed_time_limit_mins')
+    ordering = ('user', 'exam', 'attempt_number',)
```

Comment:
because we're going to have a ton of records here and we usually only care about the recent ones we should probably just order by created or updated. I think that's actually the default but would have to check

Reply:
okay looks like it's not the default but I'd recommend most recently updated ('-modified')